css/v202209/symbiota/main.css

```diff
@@ -233,11 +233,25 @@ hr {
   background-color: var(--darkest-color);
 }
 
+.button-danger {
```

Comment:
I think this should be in base.css. Actually, probably in both base.css files. Right, @egbot ?